lokalutvikling/postgres-init/inntektsmelding.sql

```diff
@@ -1,5 +1,5 @@
-CREATE DATABASE inntektsmelding;
```

Comment:
Antar det var meningen å committe dette? Det ødelegger vel ikke for kjøring i docker

Reply:
ja, det er for å få kjørt opp applikasjonen lokalt med lokal database